src/main/java/tools/jackson/databind/json/JsonMapper.java

```diff
@@ -142,17 +146,44 @@ public JsonMapper(Builder b) {
     /**********************************************************************
      */
 
-    public static JsonMapper.Builder builder() {
+    public static Builder builder() {
         return new Builder(new JsonFactory());
     }
 
     public static Builder builder(JsonFactory streamFactory) {
         return new Builder(streamFactory);
     }
 
+    /**
+     * Modifies the settings of this builder to more closely match the default configs used
+     * in Jackson 2.x versions.
+     * <p>
+     *     This method is still a work in progress and may not yet fully replicate the
+     *     default settings of Jackson 2.x.
+     * </p>
+     */
+    public static Builder builderWithJackson2Defaults() {
```

Comment:
Let's change this same way as `JsonFactoryBuilder`, to allow override by other format mappers (`XmlMapper` etc)